modules/weko-index-tree/weko_index_tree/utils.py

```diff
@@ -118,26 +118,49 @@ def get_tree_json(index_list, root_id):
         index_relation[index_element.pid].append(index_element.cid)
         index_position[index_element.cid] = position
 
+    def get_user_list_expand():
+        """Get list index expand."""
+        key = current_app.config.get(
+            "WEKO_INDEX_TREE_STATE_PREFIX",
+            WEKO_INDEX_TREE_STATE_PREFIX
+        )
+        return session.get(key, [])
+
     def generate_index_dict(index_element, is_root):
         """Formats an index_element, which is a tuple, into a nicely formatted dictionary."""
         index_dict = index_element._asdict()
+
         if not is_root:
             pid = str(index_element.pid)
             parent = index_list[index_position[index_element.pid]]
             while parent.pid and parent.cid != root_id:
                 pid = '{}/{}'.format(parent.pid, pid)
                 parent = index_list[index_position[parent.pid]]
             index_dict.update({'parent': pid})
+
+        list_index_expand = get_user_list_expand()
+        is_collapsed_on_init = str(index_element.cid) in list_index_expand
         index_dict.update({
             'id': str(index_element.cid),
             'value': index_element.name,
             'position': index_element.position,
             'emitLoadNextLevel': False,
-            'settings': {'isCollapsedOnInit': True, 'checked': False}
+            'settings': {
+                'isCollapsedOnInit': not is_collapsed_on_init,
```

Comment:
I couldn't figure why `isCollapsedOnInit` != `is_collapsed_on_init`. Both have the same name, so I was thinking they should hold the same value. Please kindly explain.